frontend/src/scenes/userLogic.tsx

```diff
@@ -111,11 +119,12 @@ export const userLogic = kea<userLogicType<UserType, EventProperty, UserUpdateTy
 
                         posthog.register({
                             posthog_version: user.posthog_version,
-                            has_slack_webhook: !!user.team.slack_incoming_webhook,
+                            has_slack_webhook: !!user.team?.slack_incoming_webhook,
                         })
                     }
                 }
-            } catch {
+            } catch (e) {
+                console.error(e)
```

Comment:
?

Reply:
This is intentional, it was hard to debug this since the error was just eaten up, I'd rather it went to the console in addition to being handled by logic.